### DISCUSSION TITLE
Reports : New installs

### DISCUSSION THREAD

**Discussion by mfenton01** [opendcim/openDCIM#1441] [Q&A] [answered]
I have 3  items to mention here .  

 When running reports from the cli  I get PHP Warning:  Undefined property: Device::$Reservation in /opt/openDCIM/openDCIM-23.02/report-em_new_installs.php on line 61 .  This could be normal  ? 

Concerning the report, I provided image .   In the header it shows Last 7 days  .   In the paragraph it indicates 30 Days . Is there a switch I can use to make it the Last  30 days or better yet Last <n>  days ?  In the configuration new installs is set to 30 still only get 7 

I need ability to send  to other  individual email  or distribution email addresses . 

RHEL - 8.8
MySQL - 10.3.35-MariaDB
PHP - 8.0.27

**Comment by wilpig** [comment, marked as answer]
https://github.com/opendcim/openDCIM/commit/67f1a9602e4baee7860dc85a9a437d102e509572
